torchx/schedulers/ray_scheduler.py

```diff
@@ -132,7 +144,44 @@ def run_opts(self) -> runopts:
         return opts
 
     def schedule(self, dryrun_info: AppDryRunInfo[RayJob]) -> str:
-        raise NotImplementedError()
+        # TODO: What to do with cfg
+        cfg: RayJob = dryrun_info.request
+
+        # TODO: figure out how to use
+        # app_id not needed here - ray will return a job_id
+        app_id: str
+
+        # no cluser config is needed since users will call ray up
+        cluster_config_file: str
+
+        # can set this in driver script
+        cluster_name: Optional[str] = None
+        
+        # not sure what to do with this
+        verbose: bool = False
+
+        # not needed?
+        cfg.copy_script_dirs = True
```

Comment:
We still need them; otherwise, we can't now which files or directories to specify as part of `runtime_env`.